docs/understanding-your-pipeline/data-products/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Data Products Introduction"
+sidebar_position: 7
+sidebar_label: "🆕 Data Products Introduction"
+description: "Snowplow's data products enable organizations to easily generate AI and BI-ready data that is reliable, clear, compliant, accurate, and predictable"
+---
+
+Data products are Snowplow’s solution to helping organisations more easily create & democratise data that is AI & BI ready; data that is reliable, explanatory, compliant, accurate, and predictable. 
```

Comment:
Sorry, can we replace this with: 
Data products are Snowplow's solution to helping organizations more easily create and democratise behavioural data. By creating well-documented datasets, that are both human and machine readable, you can more easily collaborate around data & unlock self-service analytics at scale.

Reply:
done